WcaOnRails/app/models/competition_payment_integration.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# TODO: Add validation to ensure that multiple accounts of the same type don't get added to CompetitionPaymentIntegration
+class CompetitionPaymentIntegration < ApplicationRecord
+  belongs_to :connected_account, polymorphic: true
+
+  belongs_to :competition
+
+  AVAILABLE_INTEGRATIONS = {
+    'paypal' => 'ConnectedPaypalAccount',
```

Comment:
Using arrow notations in hashes is considered "old fashion" and I'm surprised RuboCop doesn't get all sassy about it.
```suggestion
    paypal: 'ConnectedPaypalAccount',
```

Reply:
Pinging, as you didn't seem to address this in your most recent commit